content/millennium/r4/diagnostic/observation.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Observation | R4 API
+---
+
+# Observation
+
+* TOC
+{:toc}
+
+## Overview
+The Observation resource provides measurements or simple assertions about a patient that are useful for establishing baselines or trends, monitoring a patient's progress, and establishing diagnoses. Most observations are simple name/value pair assertions but some observations, such as blood pressure, group other observations together logically. Examples of common observations are: Laboratory results (blood sugar, hemoglobin), Vital signs (temperature, blood pressure), Personal characteristics (height, weight), and Social history (tobacco/alcohol use, employment status). Pathology reports, radiology reports, and other textual reports should be represented by the DiagnosticReport resource.
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
```

Comment:
all these links need to be https://

Reply:
https://github.com/cerner/fhir.cerner.com/pull/345/commits/85bb1a56bd4635bc292f0fbe1300685b9970981e#diff-d42a2575832583fdd3d20dff4a54f294R15